src/classes/hotel.js

```diff
@@ -0,0 +1,110 @@
+import Customer from './customer';
+import Room from './room';
+import Booking from './booking';
+import roomsData from '../../test/test-data/room-data';
+
+class Hotel {
+  constructor(roomsData, customersData, bookingsData) {
+    this.roomsData = roomsData;
```

Comment:
These first 3 properties (this.roomsData, this.customersData, and this.bookingsData) don't have tests. I understand why you don't have test for them, but it's my understanding that we have to have tests for every property.